matterhorn.cabal

```diff
@@ -119,7 +119,8 @@ executable matterhorn
                      , skylighting          >= 0.3.3.1 && < 0.4
                      , timezone-olson       >= 0.1.7   && < 0.2
                      , timezone-series      >= 0.1.6.1 && < 0.2
-                     , aeson                >= 1.2.3.0 && < 1.3
```

Comment:
Also, was there a reason for changing the Aeson bounds?

Reply:
On a related note, we haven't documented this really explicitly, but we tend to use tight bounds on all dependencies to ensure build stability. We aren't optimizing for allowing the average Hackage user to build Matterhorn against a wide variety of dependency versions. If there are ever changes to the dependency bounds, they need to be well-motivated and well-tested: if we change bounds because we need to, the commit should provide the justification, and if we increase upper bounds to permit new versions, we should also increase lower bounds to ensure that the version space continues to be constrained.

Reply:
This pull request doesn't seem to change anything related to Aeson functionality, so if you wanted to change the Aeson bounds, I'd like to see that in a separate commit or pull request.

Reply:
This was by mistake. I reverted it.

Reply:
Okay, thank you!